server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/WhiteListManager.java

```diff
@@ -1,793 +0,0 @@
-/****************************************************************
- * Licensed to the Apache Software Foundation (ASF) under one   *
```

Comment:
Also removals shall be documented as a breaking change in upgrade-instructions.md with an alternative....